recipe/meta.yaml

```diff
@@ -23,12 +22,12 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - setuptools
```

Comment:
Should we consider building with `cython`? See https://github.com/fonttools/fonttools/blob/4.49.0/setup.py#L38-L67